app/shared/db/misc.server.ts

```diff
@@ -32,7 +32,7 @@ export const publish = async ({
     previousTreeId: project.devTreeId,
     nextTreeId: tree.id,
   });
-  const { prodTreeIdHistory } = project;
+  const prodTreeIdHistory = project.prodTreeIdHistory as unknown as string[];
```

Comment:
hmm, maybe we should refine the type closer to the source?

Reply:
Fixed!